test/TestExpectations.json

```diff
@@ -3623,6 +3623,12 @@
     "parameters": ["firefox", "headful", "webDriverBiDi"],
     "expectations": ["FAIL"]
   },
+  {
+    "testIdPattern": "[launcher.spec] Launcher specs Puppeteer Browser.disconnect should reject navigation when browser closes",
```

Comment:
See this a lot in the past 3 days marking as flaky:
https://github.com/puppeteer/puppeteer/actions/runs/6878600308/job/18708643595?pr=11397